src/components/Spinner.tsx

```diff
@@ -42,7 +42,7 @@ const Spinner = ({
           <ImgixImage
             source={SpinnerImageSource as any}
             style={style}
-            tintColor={color ?? colors.whiteLabel}
```

Comment:
This was changed some time ago, but the default value is an empty string, in this case `||` should be used since it takes falsy values and not only null-ish values